ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1288 @@
+{
```

Comment:
<p>Why do you need this section? You always test the model against full test set before saving.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>It's simple test.</p><p>Notebook save 2 files. <code>metrics.csv</code> and <code>model.pt</code></p><p>We checked that both files have expected behavior in this block.</p>